algebra-core/src/main/scala/cats/algebra/Priority.scala

```diff
@@ -0,0 +1,66 @@
+package cats
+package algebra
+
+/**
+ * Priority is a type class for prioritized implicit search.
+ *
+ * This type class will attempt to provide an implicit instance of `P`
+ * (the preferred type). If that type is not available it will
+ * fallback to `F` (the fallback type). If neither type is available
+ * then a `Priority[P, F]` instance will not be available.
+ *
+ * This type can be useful for problems where multiple algorithms can
+ * be used, depending on the type classes available.
+ */
+sealed trait Priority[+P, +F] {
```

Comment:
It looks like this trait is never actually used, so I think we should scrap it.

Reply:
Funny, I make good use of it in my own project and maybe it even got submitted to Scala? https://github.com/typelevel/algebra/issues/67

Anyway, scrapped.